doc/ansible/dev/developer_guide.md

```diff
@@ -211,7 +211,7 @@ Create a Custom Resource Definition (CRD) and proper Role-Based Access Control
 (RBAC) definitions for resource Foo. `operator-sdk` auto-generates these files
 inside of the `deploy` folder:
 ```bash
-$ kubectl create -f deploy/crds/foo.example.com_foos_crd.yaml
+$ kubectl create -f deploy/crds/foo_v1alpha1_foo_crd.yaml
```

Comment:
The latest released creates the files with these names and not so long as `foo_v1alpha1_foo_cr` . See:

```
$ operator-sdk new --type ansible --kind Foo --api-version foo.example.com/v1alpha1 foo-operator
INFO[0000] Creating new Ansible operator 'foo-operator'. 
INFO[0000] Created deploy/service_account.yaml          
INFO[0000] Created deploy/role.yaml                     
INFO[0000] Created deploy/role_binding.yaml             
INFO[0000] Created deploy/crds/foo.example.com_foos_crd.yaml 
INFO[0000] Created deploy/crds/foo.example.com_v1alpha1_foo_cr.yaml 
```